### ISSUE TITLE
Set up automated accessibility checking in CI

### ISSUE THREAD

**Issue by thatbudakguy** [sul-dlss/mylibrary#951]
The approach we want to borrow is the one @cbeer implemented in blacklight; see: [Test a11y with aXe (#2478)](https://github.com/projectblacklight/blacklight/commit/cb14811a0acbca8f3da4afa6fb601470c53476b8)

The idea behind testing this here (first) is that mylibrary isn't covered by SiteImprove because it's behind auth, so the onus is more on us to catch a11y issues. If the setup is easy enough and relatively pain-free, we can duplicate it across other projects.

**Comment by dnoneill** [comment]
closed with https://github.com/sul-dlss/mylibrary/pull/1066
